tests/ansi/alloc.c

```diff
@@ -18,6 +18,8 @@ int main() {
 	p = aligned_alloc(1, 1);
 	assert(p != NULL);
 
+	// It seems that glibc doesn't report error in these cases.
```

Comment:
We should probably submit a bug report then...

Reply:
Meh... it'd probably break some programs if they changed that behaviour. We'd still have to work around it here anyway, so we need to decide on a solution.